integration-test/src/java/com/twitter/heron/integration_test/topology/serialization/resources/SerializationTopology.json

```diff
@@ -0,0 +1 @@
+[10]
```

Comment:
@billonahill I just pushed a commit that asserts the data itself in bolts.
Also, CustomSpout is wrapped by IntegrationTestSpout when TestTopologyBuilder creates a topology.